.github/workflows/perf.yaml

```diff
@@ -40,6 +40,9 @@ jobs:
           client-id: ${{ secrets.AZURE_CLIENT_ID }}
           tenant-id: ${{ secrets.AZURE_TENANT_ID }}
           subscription-id: ${{ secrets.AZURE_SUBSCRIPTION }}
+        
+      - id: fetch-latest-github-release
+        uses: cityoflosangeles/fetch-latest-github-release@v2.0.4
```

Comment:
Did you test that this works as expected?

Reply:
Can we use GitHub API instead? 

```sh
curl -s https://api.github.com/repos/microsoft/retina/releases | jq -r '.[0].name'
```